apps/bare-expo/e2e/utils/report.js

```diff
@@ -0,0 +1,35 @@
+import chalk from 'chalk';
+
+export function reportMagic({ testName, failed, results, failures }) {
+  const formatResults = results =>
+    results &&
+    results
+      // Remove random "undefined" from beginning
```

Comment:
Figure out why this is happening and remove it at the source instead of trying to undo its effect after the fact

Reply:
probably better for another PR. Still plenty of work to do on this bare-expo runner